client/packages/invoices/src/Returns/modals/InboundReturn/ReturnQuantitiesTable.tsx

```diff
@@ -15,35 +20,72 @@ export const QuantityReturnedTableComponent = ({
     line: Partial<GeneratedInboundReturnLineNode> & { id: string }
   ) => void;
 }) => {
+  // what to do here if there are multiple items? Some might have PV and others not...
+  // const { packVariantExists } = usePackVariant(item?.id || '', null);
```

Comment:
From what I can see, the pack variant implementations so far take the itemId, and if there are pack variants set for that item, we set the column as a unit-variant-and-pack-size column, and if not, then the usual pack-size column. 

That would work for the add by item/edit line use cases, but when adding from outbound shipment, we can have lots of items... some might have pack variants configured, and some might not. How should we display this column then?

Reply:
Good question, maybe discuss tomorrow in stand-up?

Reply:
@andreievg does this come up anywhere else?

Reply:
Ouch, nope this is something new, I think fir right now we can just have the `label.punit-variant-and-pack-size` label regardless

 Nice one, for already finding the pack variant stuff

Reply:
Ok cool have put that label in, and added a note in another issue [here](https://github.com/msupply-foundation/open-msupply/issues/3212#issuecomment-2008005859) to think more about how we might display this...